src/components/TheTransactionList.vue

```diff
@@ -9,7 +9,29 @@
       <div class="modal-background"></div>
       <div class="modal-card">
         <header class="modal-card-head">
-          <p class="modal-card-title is-title-shrink">Toutes les opérations</p>
+          <p class="modal-card-title is-title-shrink">
+            <span class="ml-2">Toutes les opérationssss</span
```

Comment:
Why several "S" ? ^^'

Reply:
Test on iPhone

- I can launch export
- I can find the file in "On My iPhone/Monujo/Transactions.csv

https://user-images.githubusercontent.com/94766388/189350440-c0f54f65-408c-4a93-b3b8-90c11132385d.MP4

The exported file is the same as the one exported via the web interface
[TransactionsIPhone.csv](https://github.com/Lokavaluto/monujo/files/9535404/Transactions.csv)
[TransactionsWeb.csv](https://github.com/Lokavaluto/monujo/files/9535407/TransactionsWeb.csv)

I want to clarify that this way of working is not at all intuitive for an iPhone user. I've never had to go into an application's folder before.
I asked around it's the same.
For me this feature is unusable in practice.

The application would have to offer a way to somehow visualize the existence of the file with a share button to be able to do something with that file with a "share" button.

I would recommend not to provide this functionality on iPhone as it is.

Reply:
thanks for the great quality of your review, as you suggested, we are heading for only giving the sharing capability on iphone.